config.md

```diff
@@ -265,6 +265,19 @@ Annotations are key-value maps.
 }
 ```
 
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
```

Comment:
On Wed, May 11, 2016 at 02:12:16AM -0700, James Hunt wrote:
> > +ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
>
> imagePath needs to be absolute since it will not form part of the
> bundle, so the runtime needs to know exactly where on the host to
> find it.

With Linux-namespace containers, the runtime knows exactly where to
find root.path without it having to be absolute (relative paths are
interpreted relative to the directory containining config.json).  So I
don't see a reason to *require* absolute paths.  Then folks could put
the image in the bundle if they wanted, or point at an absolute path
if they wanted.